### DISCUSSION TITLE
user previlege to access frigate

### DISCUSSION THREAD

**Discussion by acmeGR** [blakeblackshear/frigate#982] [Ask A Question]
I have an ipad on my wall and I have created a user without administrator privilages for it. 
How can I give access to this user to be able to see frigate on sidebar?

**Comment by Apeiv** [comment]
I have the same problem

**Reply by s24dor** [reply]
The new frigate card should offer access to clips etc without access to the frigate ui

**Reply by Apeiv** [reply]
ok thanks, i installed this addon but show any clips or snapshoot, and there isn't other funtions link brideye
@acmeGR do you resolve your problem?
